03/app.js

```diff
@@ -7,12 +7,64 @@ const c = randomNumber(min, max);
 
 console.log(a, b, c);
 
+function randomNumber(min, max) {
+    return Math.round((Math.random() * (max - min)) + min);
+}
 
+function getSum(a,b,c) {
+    a = Math.round(a);
+    b = Math.round(b);
+    c = Math.round(c);
+    let firstMax = 0;
+    let secondMax = 0;
+    if(a>=b || a>=c){
+        firstMax = a;
+        console.log('Przypisuje pierwszy max do ' + a);
+    }
+    if(b>=a || b>=c) {
+        if(firstMax == 0){
+            firstMax = b;
+            console.log('Przypisuje pierwszy max do ' + b);
+        } else {
+            secondMax = b;
+            console.log('Przypisuje drugi max do ' + b);
+        }
+    }
+    if(c>=a || c>=b){
+        secondMax = c;
+        console.log('Przypisuje drugi max do ' +c);
+    }
```

Comment:
To rozwiązanie może nie zadziałać jeśli będą przekazane wartości ujemne.
Najprościej byłoby utworzyć tablicę, która składa się z podanych parametrów, posortować ją malejąco i mamy największe wartości - prosto i czytelnie :)